common.yml

```
@@ -641,6 +641,37 @@ functions:
               --package-file-suffix "" \
               release.msi
 
+  "sign linux packages":
+    - command: shell.exec
+      type: system
+      params:
+        working_dir: src/github.com/mongodb/mongo-tools/
+        silent: true
+        script: |
+          echo "${signing_auth_token_server_4_4}" > ./signing_auth_token
+    - command: shell.exec
+      type: system
+      params:
+        working_dir: src/github.com/mongodb/mongo-tools/
+        silent: true
+        script: |
+          /usr/local/bin/notary-client.py \
+              --key-name "server-4.4" \
+              --auth-token-file ./signing_auth_token \
+              --comment "Evergreen Automatic Signing (mongo-tools)" \
+              --notary-url http://notary-service.build.10gen.cc:5000 \
+              --skip-missing \
+              --package-file-suffix "" \
```

```suggestion
              --package-file-suffix "" \
              --outputs "sig" \
```
This was added to the msi signing in a recent commit on master, and we should do the same here. It makes sure that we only sign the package, and don't generate other extraneous artifacts